ngraph/frontend/onnx_import/src/onnx.cpp

```diff
@@ -44,6 +44,8 @@ namespace ngraph
                 {
                     function->get_output_op(i)->set_friendly_name(
                         graph.get_outputs().at(i).get_name());
+                    function->get_output_op(i)->get_output_tensor(0).set_names(
```

Comment:
I think we don't need this change. Because OV doesn't use output tensors for result operations.
And from my point of view after this change we will have the next situation:
```
 ___________________
|   Some operation  |
|___________________|
            |
            |    Tensor name: outTensor
           \/
 ___________________
|      Result       |
|  name: outTensor  |
|___________________|
            |
            |    Tensor name: outTensor
           \/
```
I mean that tensor names are not unique, because we use one tensor name for two nGraph tensors.

Reply:
As mentioned above I will remove this code.